code/randomized_algorithms/src/README.md

```diff
@@ -1,4 +1,4 @@
-# cosmos
-Your personal library of every algorithm and data structure code that you will ever encounter
+# Randomized Algorithms
```

Comment:
Here too, you need more info. Also keep the deleted part at the bottom of the file.